### PR TITLE
Added intermediate example trajectory and abstracted thrust_scalar to launch file

### PR DESCRIPTION
**Pull request by hildebrandt-carl**
1) Changed the circle example trajectory to aggressive_circle (it's terrifying to do in a cage), and added a new circle trajectory that is significantly slower, and possibly a better intermediate stage for testing the setup. Thus the example's provided are now:

* Hover
* Circle
* Aggressive_Circle

2) Added the thrust_scalar parameter to the launch file, as it's a parameter people will need to change frequently with new drones.

3) Miscellaneous stuff e.g. moving type before value to keep consistency across entire launch file, etc.

### CODE REVIEW

File: freyja_trajectory_provider/src/trajectory_provider.cpp [hunk 1]
```diff
@@ -57,6 +57,28 @@ TrajRef getCircleReference( const ros::Duration &cur_time )
...
+  ref_state.vd = 0.0;
+  
+  ref_state.yaw = 0.0;
+
+  // set an, ae, ad to second derivatives if needed for FF..
+  return ref_state;
+}
+
+// CIRCLE: pn = A*sin(wt), pe = A*cos(wt), vn = A*w*cos(wt) ..
+TrajRef getAggressiveCircleReference( const ros::Duration &cur_time )
```
Comment (ajshank) [inline freyja_trajectory_provider/src/trajectory_provider.cpp:75]: Make common function: `getCircleReference( const ros::Duration &t, const int agg_level)`

Then, based on `agg_level` (1,2,3..) define values for variables `A` and `w`. `A` is amplitude (radius), `w` angular rate such that `2pi/w` = (seconds for one rev). So, `w=1` means 6.24s, `w=2` means 3.14s etc .. larger `w` ==> faster circles.

Then,
```
pn = A*std::sin( w*t );
pe = A*std::cos( w*t );
: 
vn = A*w*std::cos( w*t );
ve = -A*w*std::sin( w*t );
```

File: freyja_trajectory_provider/src/trajectory_provider.cpp [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
   EXAMPLE FILE; ONLY FOR SUPPORT.
 
-  Whatever you do here, output a time-based continous function to follow.
+  Whatever you do here, output a time-based continuos function to follow.
```
Comment (ajshank) [inline freyja_trajectory_provider/src/trajectory_provider.cpp:5]: "continuous" :)

File: freyja_controller.launch [hunk 3]
```diff
@@ -14,6 +14,7 @@
 	<!-- use "device:baudrate" for apm, "device" for asctec -->
 	<arg name="comm_port" default="/dev/ttyUSB0:57600"/>
 	<arg name="total_mass" default="0.95"/>
+  <arg name="thrust_scalar" default="200.0"/>
```
Comment (ajshank) [inline freyja_controller.launch:17]: Typo: `thrust_scaler`

**Review by ajshank — CHANGES_REQUESTED** [review]
Create a different branch for these changes, and let's merge that branch into `master` here.

**Comment by hildebrandt-carl** [comment]
I am closing this pull request to rebase so that not all 6 commits are present in this pull. I will make the requested changes.

**Pull request closed without merging**